### PR TITLE
Denylist and allowlist translations

### PR DESCRIPTION
**Pull request by Chexet48**

### CODE REVIEW

File: src/main/resources/assets/securitycraft/lang/it_it.json [hunk 1]
```diff
@@ -880,7 +880,7 @@
 	"option.securitycraft.keypad.isAlwaysActive": "Sempre attivo: %s",
 	"option.securitycraft.keypad.isAlwaysActive.description": "La tastiera deve rimanere attiva e sbloccata?",
 	"option.securitycraft.keypad.sendMessage": "Messaggio: %s",
-	"option.securitycraft.keypad.sendMessage.description": "Questa tastiera manda un messaggio all'attivazione (usando moduli lista bianca/nera)?",
+	"option.securitycraft.keypad.sendMessage.description": "Questa tastiera manda un messaggio all'attivazione (usando moduli lista consentiti/nera)?",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/it_it.json:883]: Still got "nera" here.

File: src/main/resources/assets/securitycraft/lang/it_it.json [hunk 2]
```diff
@@ -902,31 +902,31 @@
 	"option.securitycraft.alarm.delay": "Ritardo: %s",
 	"option.securitycraft.alarm.delay.description": "Se un allarme viene attivato, quanti secondi ci sono tra due suoni d'allarme?",
 	"option.securitycraft.keycard_reader.sendMessage": "Messaggio: %s",
-	"option.securitycraft.keycard_reader.sendMessage.description": "Questo lettore di tessere magnetiche manda un messaggio all'attivazione (usando moduli lista bianca/nera)?",
+	"option.securitycraft.keycard_reader.sendMessage.description": "Questo lettore di tessere magnetiche manda un messaggio all'attivazione (usando moduli lista consentiti/nera)?",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/it_it.json:905]: Still got "nera" here.

File: src/main/resources/assets/securitycraft/lang/it_it.json [hunk 3]
```diff
@@ -902,31 +902,31 @@
...
 	"option.securitycraft.alarm.delay.description": "Se un allarme viene attivato, quanti secondi ci sono tra due suoni d'allarme?",
 	"option.securitycraft.keycard_reader.sendMessage": "Messaggio: %s",
-	"option.securitycraft.keycard_reader.sendMessage.description": "Questo lettore di tessere magnetiche manda un messaggio all'attivazione (usando moduli lista bianca/nera)?",
+	"option.securitycraft.keycard_reader.sendMessage.description": "Questo lettore di tessere magnetiche manda un messaggio all'attivazione (usando moduli lista consentiti/nera)?",
 	"option.securitycraft.keycard_reader.signalLength": "Lunghezza segnale: %s",
 	"option.securitycraft.keycard_reader.signalLength.description": "Quanto dovrebbe durare un segnale di redstone dall'attivazione? (In tick, 20 tick = 1 secondo)",
 	"option.securitycraft.keypad_chest.sendMessage": "Messaggio: %s",
-	"option.securitycraft.keypad_chest.sendMessage.description": "Questo baule manda un messaggio all'attivazione (usando moduli lista bianca/nera)?",
+	"option.securitycraft.keypad_chest.sendMessage.description": "Questo baule manda un messaggio all'attivazione (usando moduli lista consentiti/nera)?",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/it_it.json:909]: Still got "nera" here.

File: src/main/resources/assets/securitycraft/lang/it_it.json [hunk 4]
```diff
@@ -902,31 +902,31 @@
...
+	"option.securitycraft.keycard_reader.sendMessage.description": "Questo lettore di tessere magnetiche manda un messaggio all'attivazione (usando moduli lista consentiti/nera)?",
 	"option.securitycraft.keycard_reader.signalLength": "Lunghezza segnale: %s",
 	"option.securitycraft.keycard_reader.signalLength.description": "Quanto dovrebbe durare un segnale di redstone dall'attivazione? (In tick, 20 tick = 1 secondo)",
 	"option.securitycraft.keypad_chest.sendMessage": "Messaggio: %s",
-	"option.securitycraft.keypad_chest.sendMessage.description": "Questo baule manda un messaggio all'attivazione (usando moduli lista bianca/nera)?",
+	"option.securitycraft.keypad_chest.sendMessage.description": "Questo baule manda un messaggio all'attivazione (usando moduli lista consentiti/nera)?",
 	"option.securitycraft.keypad_furnace.sendMessage": "Messaggio: %s",
-	"option.securitycraft.keypad_furnace.sendMessage.description": "Questa fornace manda un messaggio all'attivazione (usando moduli lista bianca/nera)?",
+	"option.securitycraft.keypad_furnace.sendMessage.description": "Questa fornace manda un messaggio all'attivazione (usando moduli lista consentiti/nera)?",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/it_it.json:911]: Still got "nera" here.

File: src/main/resources/assets/securitycraft/lang/it_it.json [hunk 5]
```diff
@@ -936,7 +936,7 @@
 	"option.securitycraft.scanner_door.signalLength": "Tempo prima di chiudersi: %s",
 	"option.securitycraft.scanner_door.signalLength.description": "Quanto dovrebbe stare aperta la porta? (In tick, 20 tick = 1 secondo. Imposta a 0 per far rimanere aperta la porta)",
 	"option.securitycraft.keypad_door.sendMessage": "Messaggio: %s",
-	"option.securitycraft.keypad_door.sendMessage.description": "Questa porta con tastiera manda un messaggio all'attivazione (usando moduli lista bianca/nera)?",
+	"option.securitycraft.keypad_door.sendMessage.description": "Questa porta con tastiera manda un messaggio all'attivazione (usando moduli lista consentiti/nera)?",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/it_it.json:939]: Still got "nera" here.

**Review by bl4ckscor3 — CHANGES_REQUESTED** [review]
Just a handful of missed things, otherwise this is good! Many thanks!

File: src/main/resources/assets/securitycraft/lang/it_it.json (hunk 5 above)
Comment (Chexet48) [reply src/main/resources/assets/securitycraft/lang/it_it.json]: Now it should be fine!

**Comment by Chexet48** [comment]
> 
> Just a handful of missed things, otherwise this is good! Many thanks!

Now it should be fine!

**Review by bl4ckscor3 — APPROVED** [review]

**Pull request merged**